org/branchprotection.yml

```diff
@@ -81,3 +81,32 @@ branch-protection:
             required_approving_review_count: 1
             bypass_pull_request_allowances:
               teams: ["wg-app-runtime-platform-bots"]
+
+        # CF-on-K8s
+        korifi-ci:
+          protect: false
+        cf-k8s-secrets:
+          protect: false
+        korifi:
+          protect: true
+          enforce_admins: true
+          allow_force_pushes: false
+          allow_deletions: false
+          allow_disabled_policies: true
+          include: ["^main$"]
+          required_status_checks: 
+            checks:
```

Comment:
The syntax of required_status_checks is not correct.
Doc: https://docs.prow.k8s.io/docs/components/optional/branchprotector/ 
See e.g. configuration of pcap-release above

Reply:
We went for this syntax as the [Github docs](https://docs.github.com/en/rest/branches/branch-protection?apiVersion=2022-11-28#update-branch-protection) mark `contexts` as deprecated - and thought it was ok since the checks passed?

Reply:
I would say that the branchprotector tool still requires the `contexts` node: https://github.com/kubernetes/test-infra/blob/master/prow/github/types.go#L633

The `org-config-generation-check (pull_request)` includes a schema validation for branchprotection.yml but the schema includes only the fields that are needed by the org automation. It doesn't include all the detailed fields supported by the branchprotector tool. This could be improved. Problem is that there is little documentation available and would have to reverse engineer the schema from the branchprotector sources - but I could start with the fields that are used so far in branchprotection.yml and extend case by case.
For now, the final validation comes when this PR got merged...